File: api/v2010/message/create-test-sms-messages-example-2/create-test-sms-messages-example-2.5.x.php
<?php

// Update the path below to your autoload.php,
// see https://getcomposer.org/doc/01-basic-usage.md
require_once '/path/to/vendor/autoload.php';

use Twilio\Rest\Client;

// Your Account Sid and Auth Token from twilio.com/console
$sid    = "ACaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa";
$token  = "your_auth_token";
$twilio = new Client($sid, $token);

$message = $twilio->messages
                  ->create("+15005550009",
                           array(
                               'body' => "Hey Mr Nugget, you the bomb!",
                               'from' => "+15005550006"
                           )
                  );

print($message.sid);

File: api/v2010/message/create-test-sms-messages-example-2/create-test-sms-messages-example-2.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using Twilio;
using Twilio.Rest.Api.V2010.Account;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var message = MessageResource.Create(
            body: "Hey Mr Nugget, you the bomb!",
            from: new Twilio.Types.PhoneNumber("+15005550006"),
            to: new Twilio.Types.PhoneNumber("+15005550009")
        );

        Console.WriteLine(message.Sid);
    }
}
